tests/integration/components/bs-dropdown/menu/link-to-test.js

```diff
@@ -64,32 +60,30 @@ module('Integration | Component | bs-dropdown/menu/link-to', function (hooks) {
     });
   });
 
-  if (supportsAngleBracketsLinkTo) {
-    module('<LinkTo> properties', function () {
-      test('simple route link', async function (assert) {
-        await render(hbs`<BsDropdown::menu::link-to @route="index">Link</BsDropdown::menu::link-to>`);
+  module('<LinkTo> properties', function (hooks) {
+    setupNoDeprecations(hooks);
```

Comment:
The diff looks large, but basically I just moved that `setupNoDeprecations` call from the outer `module()` to this submodule, thus not applying it to the module testing positional params.